runtime/compiler/compile/J9SymbolReferenceTable.cpp

```diff
@@ -1583,8 +1600,22 @@ J9::SymbolReferenceTable::findOrCreateConstantDynamicSymbol(TR::ResolvedMethodSy
       }
    else
       {
-      symRef = findOrCreateCPSymbol(owningMethodSymbol, cpIndex, TR::Address, true, dynamicConst);
+      TR::KnownObjectTable::Index knownObjectIndex = TR::KnownObjectTable::UNKNOWN;
+      TR::KnownObjectTable *knot = comp()->getKnownObjectTable();
```

Comment:
Sorry if I'm missing something obvious - can you explain why the call to `getKnownObject()` is sufficient for the constant dynamic case, while the method handle cases use `getOrCreateKnownObjectTable()`?